_posts/2022-02-07-zero-six-one-release.md

```diff
@@ -20,9 +20,9 @@ Smaller changes that you'll notice include much-awaited updates and fixes, inclu
 
 ## ☁️ Cloud Integrations: AWS and Azure
 
-You can now use [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) and [AWS' S3](https://aws.amazon.com/s3?tag=soumet-20) as your [artifact store](https://docs.zenml.io/getting-started/core-concepts) for ZenML pipelines. We implemented all the relevant `fileio` methods to enable this. If you prefer to use Amazon's AWS or Microsoft's Azure, we hope these new integrations will be the start of more options for you when using ZenML.
+You can now use [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) and [AWS' S3](https://aws.amazon.com/s3?tag=soumet-20) as your [artifact store](https://docs.zenml.io/v/0.13.2/mlops-stacks/artifact-stores) for ZenML pipelines. We implemented all the relevant `fileio` methods to enable this. If you prefer to use Amazon's AWS or Microsoft's Azure, we hope these new integrations will be the start of more options for you when using ZenML.
 
-To learn more, check out [the new documentation page](https://docs.zenml.io/v/0.6.1/features/cloud-pipelines/guide-aws-gcp-azure) we just included to guide you in deploying your pipelines to AWS, GCP and/or Azure.
+To learn more, check out [the new documentation page](https://docs.zenml.io/) we just included to guide you in deploying your pipelines to AWS, GCP and/or Azure.
```

Comment:
Wait we don't have platform guide anymore?

Reply:
I think the page which is mentioned here does not exist anymore. That is why I changed it to the main docs but I can still link it to the deployment page.

Reply:
Yeah some more specific page might be nice. Or just remove this sentence ;)